gen/calc.py

```diff
@@ -234,6 +234,11 @@ def calculate_adminrouter_auth_enabled(oauth_enabled):
     return oauth_enabled
 
 
+def validate_os_type(os_type):
+    can_be = ['coreos', 'el7']
+    assert os_type in can_be, 'Must be one of {}. Got {}'.format(can_be, os_type)
```

Comment:
@cmaloney should I include a reference to this under `entry['validate']` in `gen/calc.py` or `gen/aws/calc.py`?